docs/openoni.md

```diff
@@ -0,0 +1,242 @@
+
+##### Theme and Plugins
+Add the theme and plugins it incorporates to `INSTALLED_APPS`:
+
+```py
+# List of configuration classes / app packages in order of priority (i.e., the
+# first item in the list has final say when collisions occur)
+INSTALLED_APPS = (
+    # Default
+#    'django.contrib.admin',
+#    'django.contrib.auth',
+#    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+
+    # Plugins
+    # See https://github.com/open-oni?q=plugin for available plugins
+    'onisite.plugins.calendar',
+    'onisite.plugins.featured_content',
+    'onisite.plugins.map',
+
+    # Open ONI
+    'django.contrib.humanize',  # Added to make data more human-readable
+    'sass_processor',
+    'themes.nebraska',
+    'themes.default',
+    'core',
+)
+```
+
+##### Title and Project Name
+Set the title and project name text for the website
+
+```py
+# SITE_TITLE that will be used for display purposes throughout app
+# PROJECT_NAME may be the same as SITE_TITLE but can be used
+# for longer descriptions that will only show up occasionally
+# Example 'Open ONI' for most headers, 'Open Online Newspapers Initiative'
+# for introduction / about / further information / etc
+SITE_TITLE = "Nebraska Newspapers"
+PROJECT_NAME = "Nebraska Newspapers"
+```
+
+
+#### Logging
+Create symlink at `/var/log/openoni` to `/opt/openoni/log`
+
+```bash
+ln -s /opt/openoni/log /var/log/openoni
+```
+
+#### URLs
+Set the URLs file to use the theme and plugins it incorporates
+
+```bash
+cp onisite/urls_example.py onisite/urls.py
+```
+
+`vim onisite/urls.py`:
+```python
+# Copy this to urls.py.  Most sites can leave this as-is.  If you have custom
+# apps which need routing, modify this file to include those urlconfs.
+from django.conf.urls import url, include
+
+# Django documentation recommends always using raw string syntax: r''
+urlpatterns = [
+  # Plugin URLs
+  #url(r'^map/', include("onisite.plugins.map.urls")),
+
+  # Theme URLs
+  #url(r'', include("themes.(theme_name).urls")),
+
+  # Open ONI URLs
+  url(r'', include("core.urls")),
+]
+```
+
+
+## Compile Static Assets
+Run these commands as a regular user rather than root
+
+```bash
+cd /opt/openoni
+source ENV/bin/activate
+
+./manage.py compilescss
+./manage.py collectstatic -c
+
+# Grant write access for both Apache and group
+sudo chown -R apache static/compiled/
+sudo chmod -R g+w static/compiled/
+```
+
+In production environments, perform a graceful Apache restart after re-compiling static assets so the app uses the updated static file hash fingerprints in the URLs rendered in templates:
+
+```bash
+sudo apachectl graceful
+```
+
+## Load Batches
+Run these commands as a regular user rather than root
+
+```bash
+# Repeat as necessary
+./manage.py load_batch /opt/openoni/data/batches/(batch_name)/
+
+# Run a script with nohup in the background to ingest multiple batches quietly
```

Comment:
I would mention, perhaps, that the main purpose of nohup is to make sure that disconnecting from the server won't stop the script from running?

Reply:
Good point. I can add something to that effect here and in our theme's docs :+1: